client/state/stats/lists/actions.js

```diff
@@ -15,7 +15,7 @@ export const ALL_SITES_ID = -1;
  * @param  {number} siteId   Site ID
  * @param  {string} statType Stat Key
  * @param  {Object} query    Stats query
- * @param  {Array}  data     Stat Data
+ * @param  {Array|Object}  data     Stat Data
```

Comment:
The location data is not array, I am not changing the data format for that as that would require more changes